tests/appsec/iast/aspects/test_join_aspect_fixtures.py

```diff
@@ -458,24 +422,22 @@ def test_string_join_all_tainted(self):
             source_value="foo",
             source_origin=OriginType.PARAMETER,
             start=1,
-            len_pyobject=5,
         )
         tainted_fghi = taint_pyobject(
             pyobject="fghi",
             source_name="fghi",
             source_value="foo",
             source_origin=OriginType.PARAMETER,
             start=0,
-            len_pyobject=4,
         )
         result = mod.do_join_args_kwargs(tainted_base_string, tainted_fghi)
         assert result == "f+abcde-g+abcde-h+abcde-i"
 
         ranges = get_tainted_ranges(result)
         assert result[ranges[0].start : (ranges[0].start + ranges[0].length)] == "f"
-        assert result[ranges[1].start : (ranges[1].start + ranges[1].length)] == "abcde"
+        assert result[ranges[1].start : (ranges[1].start + ranges[1].length)] == "abcde-g"
```

Comment:
Why is this the proper behaviour? Shouldn't the range stop at `-`? `g` Is in the next range